pandas_gbq/exceptions.py

```diff
@@ -3,6 +3,14 @@
 # license that can be found in the LICENSE file.
 
 
+class GenericGBQException(ValueError):
```

Comment:
Sanity check - why is `ValueError` chosen as a base, does it really fit all use cases?

I see that other exceptions such as `AccessDenied` also inherit from `ValueError`, which feels somewhat unusual.

Reply:
It's been a ValueError since at least 2015

https://github.com/pandas-dev/pandas/pull/10857/files#diff-ff6e2a94a30a15b176a71736e407806b578e509232bdf7c04e299133a4fb3430R44

I agree that this is unusual. Not sure that we want to be changing it now. Certainly not in this PR.

Reply:
Seems like some voodoo legacy that we indeed should not touch to not awake any sleeping daemons. At least not yet. :)